pages.fr/common/vim.md

```diff
@@ -18,11 +18,11 @@
 
 - Sauvegarder et fermer :
 
-`:wq<Entrée>`
+`{{ZZ|:wq<Entrée>}}`
 
 - Annuler la dernière opération :
 
-`u`
+`<Esc>u`
```

Comment:
Inconsistency: some translations have the `Undo the last operation:`, some are missing it.